### ISSUE TITLE
Identify the non-use of the function return

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1027]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Feature request                          |

There a lot (95%?) of PHP native functions that will return some valuable data, but it make sense only if the return is used. Else the function could be safety dropped.

For instance:

```php
// Warning: value returned by strpos() is not used.
strpos($haystack, $needle);

// Valid solutions:
$strpos = strpos($haystack, $needle);
if (strpos($haystack, $needle)) {}
return strpos($haystack, $needle);
```

Things to think about:

* Map all native functions that it will be truth. Functions like `file_put_contents()` not need be stored on a variable;
* Maybe it could includes native methods too;
* Check the rarity of occurrence. If it is very uncommon, must probably will cause more CPU usage than real usefulness.

**Comment by kalessil** [comment]
Duplicates #1026
